config/packages/dev/libero_page.yaml

```diff
@@ -2,6 +2,7 @@ libero_page:
     pages:
         homepage:
             path: '/'
+            search_service: 'scholarly-articles'
```

Comment:
Been thinking about how to power the list on the homepage. eLife uses its search service, which is where our initial direction has been here. But that has been a bit problematic as it's not then easy for us to turn off search if needed (it'd be good to have a circuit breaker there). For Libero though, this needs to be a list of items. As it stands, the content API and search API lists are the same so can be interchanged. Search will be expanded to allow filtering by type etc. If we are able to keep the basic definition of how lists work the same, then it's possible to use other things here (eg could be a dedicated list). Means the config here would be a path, as opposed to the name of a service. (To use a search service, would be the path include any query string parameters, which as it stands has to be configured somewhere else.)

/cc @giorgiosironi

Reply:
Done in 30d382dabb2c158610c5bf59ac918e41e579c47b. There's already a mismatch in the dervied paths (a search service would be just `/{service-name}`, whereas as content API is `/{service-name}/items`).